public/locales/en/common.json

```diff
@@ -623,5 +629,35 @@
     "copy-link": "Copy link",
     "copy-text": "Copy text",
     "disable-share": "Cancel"
+  },
+  "personal": {
```

Comment:
Please make translations to all languages the same.

Reply:
added